plugins/janus_audiobridge.c

```diff
@@ -2866,10 +2969,12 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 						g_list_length(participant->inbuf), DEFAULT_PREBUFFERING*2);
 					participant->last_drop = now;
 				}
-				while(g_list_length(participant->inbuf) > DEFAULT_PREBUFFERING*2) {
+				while(g_list_length(participant->inbuf) > DEFAULT_PREBUFFERING) {
 					/* Remove this packet: it's too old */
 					GList *first = g_list_first(participant->inbuf);
 					janus_audiobridge_rtp_relay_packet *pkt = (janus_audiobridge_rtp_relay_packet *)first->data;
+					JANUS_LOG(LOG_ERR, "list length = %d, Remove sequence = %d\n",
+						g_list_length(participant->inbuf), pkt->seq_number);
```

Comment:
Why the LOG_ERR? This might happen a lot, and flood the logs.

Reply:
In normal situation, it doesn’t happens a lot. Now, if you want to remove it or change it ...
Today, it help to detect audio quality problem.

Reply:
I'd rather change the verbosity level.